Enable TLSv1.2 on Java 7 builds

Appveyor and Shippable builds fail.

For example [28311685](https://ci.appveyor.com/project/yegor256/rultor/builds/28311685):

``` ps
[ERROR] The build could not read 1 project -> [Help 1]
[ERROR]   
[ERROR]   The project com.rultor:rultor:2.0-SNAPSHOT (C:\projects\rultor\pom.xml) has 1 error
[ERROR]     Non-resolvable parent POM: Could not transfer artifact com.jcabi:parent:pom:0.37.1 from/to central (https://repo.maven.apache.org/maven2): Received fatal alert: protocol_version and 'parent.relativePath' points at wrong local POM @ line 35, column 13 -> [Help 2]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/ProjectBuildingException
[ERROR] [Help 2] http://cwiki.apache.org/confluence/display/MAVEN/UnresolvableModelException
Command exited with code 1
```

According to this [SO answer](https://stackoverflow.com/a/50956622) and this [sonatype blog post](https://central.sonatype.org/articles/2018/May/04/discontinued-support-for-tlsv11-and-below/) there are two solutions:
``` bash
mvn -Dhttps.protocols=TLSv1,TLSv1.1,TLSv1.2 <goals>
```
or
``` bash
export MAVEN_OPTS=-Dhttps.protocols=TLSv1,TLSv1.1,TLSv1.2
```

--- a/appveyor.yml
+++ b/appveyor.yml
@@ -19,6 +19,7 @@ install:
         [System.IO.Compression.ZipFile]::ExtractToDirectory("C:\maven-bin.zip", "C:\maven")
       }
   - cmd: SET M2_HOME=C:\maven\apache-maven-3.2.5
+  - cmd: SET MAVEN_OPTS=-Dhttps.protocols=TLSv1,TLSv1.1,TLSv1.2
   - cmd: SET PATH=%M2_HOME%\bin;%JAVA_HOME%\bin;%PATH:C:\Ruby193\bin;=%
   - cmd: SET JAVA_OPTS=-XX:MaxPermSize=2g -Xmx4g
   - cmd: mvn --version

--- a/shippable.yml
+++ b/shippable.yml
@@ -4,7 +4,7 @@ cache:
     - $HOME/.m2
 env:
   global:
-    - MAVEN_OPTS="-XX:MaxPermSize=2g -Xmx4g"
+    - MAVEN_OPTS="-XX:MaxPermSize=2g -Xmx4g -Dhttps.protocols=TLSv1,TLSv1.1,TLSv1.2"
     - JAVA_OPTS="-XX:MaxPermSize=2g -Xmx4g"
 before_install:
   - MAVEN_VERSION=3.2.5

--- a/shippable.yml
+++ b/shippable.yml
@@ -8,7 +8,7 @@ env:
     - JAVA_OPTS="-XX:MaxPermSize=2g -Xmx4g"
 before_install:
   - MAVEN_VERSION=3.2.5
-  - MAVEN_OPTS="-XX:MaxPermSize=256m -Xmx1g"
+  - MAVEN_OPTS="-XX:MaxPermSize=256m -Xmx1g -Dhttps.protocols=TLSv1,TLSv1.1,TLSv1.2"
   - wget http://mirror.cogentco.com/pub/apache/maven/maven-3/$MAVEN_VERSION/binaries/apache-maven-$MAVEN_VERSION-bin.tar.gz
   - tar -xzf apache-maven-$MAVEN_VERSION-bin.tar.gz
   - MAVEN_HOME=apache-maven-$MAVEN_VERSION

